docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-video/index.scss

```diff
@@ -0,0 +1,98 @@
+@use 'uswds-core' as *;
+
+.cgdp-video {
+	@include u-margin-x(0);
+	@include u-margin-y(0);
+	&__caption {
+		@include u-padding-top(1);
+		@include u-padding-bottom(1);
+		@include u-padding-left(2);
+		@include u-margin-top(-0.5);
+		@include u-border-left(0.5, $theme-color-secondary);
+	}
+	&__media {
+		position: relative;
+		width: 100%;
+		height: 0;
+		margin-bottom: 0;
+		overflow: hidden;
+		padding-top: 0;
+		padding-bottom: 56.25%;
+		background-color: color($theme-color-accent-cool-darker);
+
+		iframe,
+		object,
+		embed,
+		video {
+			position: absolute;
+			top: 0;
+			left: 0;
+			width: 100%;
+			height: 100%;
+		}
+	}
+}
+
+// keeping these terrible names for backwards compatibility
```

Comment:
Why are you keeping these horrible names? You are making CSS and if something else is generating that CSS, but not used by MLP, BUT someone adds it someday, you get a conflict. So if you *have* to keep it, then nest it under `.cgdp-video`.